docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -7,21 +7,15 @@
     "tags": []
```

Comment:
<p>No need to capitalize Low-Rank or Low-rank (also in other places).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>